app/src/main/kotlin/crazyboyfeng/accSettings/acc/AccHandler.kt

```diff
@@ -25,23 +26,27 @@ class AccHandler {
         val accVersionCode = resources.getInteger(R.integer.acc_version_code)
         val accVersionName = resources.getString(R.string.acc_version_name)
         try {
-            cacheAssertFile("acc_v${accVersionName}_.${accVersionCode}.tgz")
```

Comment:
It's actually a file name naming error, not a code writing error here.  
The acc distribution package is no longer named in the previous format as of 2022.1.8, so I needed to rename it manually, and an error occurred in the process.  
The new way of naming files adopted by acc is intuitive and easy to read, but it does not contain version information and cannot accurately correspond the version to the file.

My opinion is to use the old way of naming files.

Reply:
@CrazyBoyFeng, I'm open to suggestions regarding this.
Does the app parse version information just from the tarball's filename?
By the way, I have a new version ready to be released.

Reply:
@VR-25 Before installing or updating the acc, the UI application can only read the version information of the bundled acc from the filename. So I tend to use version or version number for naming.

Reply:
Ok, it'll be acc_v${accVersionName}_${accVersionCode}.tgz, so that you'll no longer have to rename the file extension.

Reply:
What about the installer's name? Is `install-tarball.sh` fine?
I would assume that due to the restricted naming rules imposed by APK packers, at least the hyphen would have been to be replaced with an underscore.
Thoughts on making things as straightforward as possible.

Reply:
The `res` resource directory (used by `AccA`) cannot use `-` and `. `, so it needs to be renamed.
Instead, I use the `asset` resource directory. It has no problems except for the incompatibility with `.gz`.

Reply:
I noticed the issue for file name and `Acc`'s name convention and also fixed the name of file but not worked at all.
Not sure How and Why but anyway.
It's better to consider the situation where `Acc` uninstalled and `AccSettings` don't know.
Or when app uninstalled,`Acc` uninstall also.

Reply:
As it is currently designed, if `AccSettings` does not detect the version of `acc`, it will automatically perform the installation in the background.

During the `acc` installation, the directory where the installation files are located will be recorded. The next time the `acc` service starts, if it finds that this directory no longer exists, then `acc` will perform a self-uninstall.